### PR TITLE
Init game with a call to the API

### PR DESCRIPTION
**Pull request by arimet**
Allows you to start the game by loading a state from an API. 
It is no longer done on the client side

### CODE REVIEW

File: api/src/board.ts [hunk 1]
```diff
@@ -13,24 +12,6 @@ enum Mode {
...
-
-export function getBoardFromFile(customPath: string): Board {
-  try {
-    const data = readFileSync(customPath, { encoding: "utf8" });
-    return JSON.parse(data).board;
-  } catch (err) {
-    return [];
-  }
-}
-
```
Comment (djhi) [inline api/src/board.ts:33]: Why did you remove those ?

**Review by djhi — CHANGES_REQUESTED** [review]

File: api/src/board.ts (hunk 1 above)
Comment (arimet) [reply api/src/board.ts:33]: I modified the type, but not the fetch inside because we modify it in the next PR

File: api/src/app.service.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { moveMarbleInDirection } from './graph';
 
 @Injectable()
 export class AppService {
-  postStartGame(playerNumber: number): GameState {
+  startGame(playerNumber: number): GameState {
```
Comment (djhi) [inline api/src/app.service.ts:9]: I think `startNewGame` returns a promise. I'm surprised TS did not yell at you

**Review by djhi — CHANGES_REQUESTED** [review]

File: api/src/app.service.ts (hunk 2 above)
Comment (djhi) [reply api/src/app.service.ts:9]: disregard, I missed the import

**Review by djhi — APPROVED** [review]
